build.gradle

```diff
@@ -350,7 +350,8 @@ dependencies {
   implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.13' // For CVE-2020-13956
   implementation group: 'org.apache.activemq', name: 'activemq-client', version: '5.16.6' // For CVE-2021-26117
   implementation group: 'org.apache.activemq', name: 'activemq-broker', version: '5.16.6' // For CVE-2021-26117
-  implementation group: 'net.minidev', name: 'json-smart', version: '2.4.7'
+  implementation group: 'net.minidev', name: 'json-smart', version: '2.4.9'
+  implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.37.2'
```

Comment:
do we need this dependency - nimbus-jose-jwt?

Reply:
It's used by something else but then the version used uses the old version of json-smart so if we don't define the dependency here for nimbus 2-23-1370 can still be exploited even though an updated version is defined